src/category_theory/differential_object.lean

```diff
@@ -25,7 +25,7 @@ namespace category_theory
 
 variables (C : Type u) [category.{v} C]
 
-variables [has_zero_morphisms C] [has_shift C]
+variables [has_zero_morphisms C] [has_shift C ℤ]
```

Comment:
Should the `\Z` be generalized here?

Reply:
I would prefer to leave that for a later PR.